### DISCUSSION TITLE
客户端看有的4k视频很卡

### DISCUSSION THREAD

**Discussion by Lzhe-DTT** [Richasy/Bili.Uwp#718] [想法]
win11，hd7850 1g显卡，看网页4k的没问题，用客户端看4k前两秒会加载不出颜色，之后会卡成幻灯片。
另外提一个小建议，希望视频播放页面那个 上次观看进度 按钮，可以弄个几秒不点击自己消失，现在我用着会一直显示到视频播放完成。谢谢开发者，很喜欢这个客户端。

**Comment by AllesUgo** [comment]
我这也是，同样的视频似乎用Windows自带的电影与电视看也会很卡，但是用传统的win32程序，包括以前Windows用的默认的播放器看都不会卡。所以我觉得这个是UWP应用本身的问题。

**Reply by Lzhe-DTT** [reply]
还真有可能，我刚试了一下官方的哔哩哔哩UWP也卡的不行。

**Reply by GDHZ5** [reply]
UWP资源占用被限制了。。。。。。

**Reply by xvzhenduo** [reply]
确实，我的都快成PPT+音频了
